### ISSUE TITLE
Possible bug with timezone ('timezone_offset')

### ISSUE THREAD

**Issue by Zazo0** [mgp25/Instagram-API#1962]
### Before submitting an issue make sure you have:
- [x] [Searched](https://github.com/mgp25/Instagram-API/search?type=Issues) the bugtracker for similar issues including **closed** ones
- [x] [Read the FAQ](https://github.com/mgp25/Instagram-API/wiki/FAQ)
- [x] [Read the wiki](https://github.com/mgp25/Instagram-API/wiki)
- [x] [Reviewed the examples](https://github.com/mgp25/Instagram-API/tree/master/examples)
- [x] [Installed the api using ``composer``](https://github.com/mgp25/Instagram-API#installation)

### Purpose of your issue?
- [x] Bug report (encountered problems/errors)
- [ ] Feature request (request for new functionality)
- [ ] Question
- [ ] Other

---

### Example Code:

/examples/accessingValues.php:

```
...
->addPost('timezone_offset', date('Z'))
...
```

Error is inside logic so there is not error log.

---

### Describe your issue

I run this code from examples and `date('Z')` always returns `0`. According to this explanation:
https://stackoverflow.com/questions/193499/utc-offset-in-php
it requires `date_default_timezone_set` to be called manually and pass to it a concrete timezone.

So in other words there are 2 cases:
1)it returns `0`
2)it returns a value for a timezone which you choose. It may be not equal to zero but anyways it becomes a constant and so you could set it directly without of calling `date('Z')`

Or maybe I don't understand the meaning of this constant, do I?

**Comment by Zazo0** [comment]
@valga ok, can I keep it always equal to zero to simplify the control of this library?

**Comment by valga** [comment]
@Zazo0 Do you live in UTC timezone? If yes, then use zero, otherwise use a real timezone offset.
